tests/scp/perform_scp.py

```diff
@@ -0,0 +1,35 @@
+import pexpect
+import sys
+
+is_source_ip = sys.argv[1] 
+ip = sys.argv[2]
+source_path = sys.argv[3]
+dest_path = sys.argv[4]
+user = sys.argv[5]
+
+command = ""
+if is_source_ip == "y":
+    command = "scp {}@{}:{} {}".format(user, ip, source_path, dest_path)
+else:
+    command = "scp {} {}@{}:{}".format(source_path, user, ip, dest_path)
+
+
+try:
+    #child = pexpect.spawn("scp root@10.250.0.102:/root/test_file.bin /home/admin")
+    child = pexpect.spawn(command)
+    prompt = child.expect(["Are you sure you want to continue connecting (yes/no)?", "password:"])
```

Comment:
This prompt was due to first time ssh into a remote host, right? It won't come up if the test was run the second time.

You might want to use ssh/scp option to skip host signature checking so that you consistently not having this step.

Reply:
If reinstall sonic to clean up DUT, then the test will fail due to mismatched hostkey. It is not easy to handle all scenarios... I agree Ying's suggestion, you can reference this: https://github.com/Azure/sonic-mgmt/blob/34b3766a4259853a71bde90a94ecbed5921e758b/tests/console/test_console_reversessh.py#L28
And the
` -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null` options should also work for scp command

Reply:
Line 20 here would have returned either 0 or 1 depending on which prompt would have appeared and the if condition would have addressed if expect returned 0. However, removing host checking seems cleaner so added it.